src/components/action-popover/action-popover.test.js

```diff
@@ -640,9 +640,13 @@ context("Test for ActionPopover component", () => {
   });
 
   describe("Check events for ActionPopover component", () => {
-    it.each([[1], [4], [6]])("should call onClick event", (element) => {
-      const callback = cy.stub();
+    let callback;
+
+    beforeEach(() => {
+      callback = cy.stub();
+    });
 
+    it.each([[1], [4], [6]])("should call onClick event", (element) => {
```

Comment:
It doesn't need to be done as part of the this PR, but it would be good if all the descriptions for the event tests were consistent. There's a few different descriptions currently and they don't really reflect what's happening here: we're triggering a click event and asserting that the expected callback is called etc

```
"should call onClick callback when a click event is triggered"
```

```
"should call onKeyDown callback when a keydown event is triggered"
```

```
"should call onBlur callback when a blur event is triggered"
```

Reply:
totally agree, updated ;)